app/presenters/api/integrations_show_presenter.rb

```diff
@@ -42,6 +53,18 @@ def environments_have_same_config?
     end
   end
 
+  def apiap?
+    provider_can_use? :api_as_product
+  end
+
+  def apicast_config_ready?
+    if apiap?
```

Comment:
Question: Is this check redundant? Wouldn't `.backend_apis.any?` return `nil` always if false?

Reply:
Not necessarily. `backend_apis.any?` may be truthy and yet `apiap? => false`.

Reply:
In fact we currently create a backend api for each service in saas and we have apiap false there :smile: